src/test/ui/async-await/issues/issue-65436-raw-ptr-not-send.stderr

```diff
@@ -6,16 +6,12 @@ LL |     assert_send(async {
    |
    = help: within `impl Future`, the trait `Send` is not implemented for `*const u8`
 note: future is not `Send` as this value is used across an await
-  --> $DIR/issue-65436-raw-ptr-not-send.rs:14:9
+  --> $DIR/issue-65436-raw-ptr-not-send.rs:14:35
    |
 LL |         bar(Foo(std::ptr::null())).await;
-   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ first, await occurs here, with `std::ptr::null()` maybe used later...
-note: `std::ptr::null()` is later dropped here
-  --> $DIR/issue-65436-raw-ptr-not-send.rs:14:41
-   |
-LL |         bar(Foo(std::ptr::null())).await;
-   |                 ----------------        ^
-   |                 |
+   |                 ----------------  ^^^^^^- `std::ptr::null()` is later dropped here
```

Comment:
This message being first is confusing, is there a way to control the order or reword it to be less confusing?

Reply:
What output would be the desirable output here?